components/NextDays.js

```diff
@@ -1,29 +1,50 @@
 import React from 'react';
-import {View, Text, StyleSheet} from 'react-native';
+import { View, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import Day from './Day';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 2,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    borderRadius: 50,
+    padding: 10,
+    backgroundColor: 'rgba(255, 255, 255, 0.2)',
+  },
+  section: {
+    flex: 1,
+    borderRightWidth: 1,
+    borderColor: '#fff',
+    justifyContent: 'center',
+    padding: 10,
+  },
+});
 
 class NextDays extends React.Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
+
+  static propTypes = {
+    days: PropTypes.array,
+  };
```

Comment:
If you are using the propsType, use it in every component and use is as in previous implementation.